content/english/_index.md

```diff
@@ -3,25 +3,29 @@
 banner:
   title: "Don't rely on Nulls, rely on us"
   content: 
-    "Allow us to make your dream a reality with our unmatchable attention to detail and elegance praising."
+    "Fear of unreliable software? Rest and allow us to make your dream a reality with our unmatched attention to detail and diligence towards reputable software"
 
 # Features
 features:
-  - title: "Culture based on Humility"
-    image: "/images/service-2.png"
-    content: "Integrity is paramount to our approach. We adhere to the highest ethical standards and craftsmanship to ensure our solutions align seamlessly with our clients' objectives. In a rapidly evolving tech landscape, we remain humble learners, valuing grounded ideas and the pursuit of improvement. Our team embraces ongoing education and exploration to shape the future of software development. Collaboration is at the heart of our ethos. We value partnerships with the open-source community, fostering an environment where diverse perspectives converge to shape tailored solutions."
+  - title: "Quality as Paramount"
+    image: "/images/service-1.png"
+    content: "In our projects, we merge practical expertise in functional programming with a commitment to delivering solutions on time and within budget --- you can trust us that your product will fly in no time. As a boutique, we focus on efficient delivery at low cost, without compromising on quality and transparency. Our team of dedicated professionals leverages both practical and abstract strategies to create reliable software solutions that meet our clients' requirements.  We value partnerships with the open-source community, fostering an environment where diverse perspectives converge to shape tailored solutions that best fit the product."
```

Comment:
The `as a boutique we <something>` feels off. It begs for a justification of why that is the case as a boutique. I would just remove it or change `boutique` for something else